src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM_DISCOUNT = 50;
```

Comment:
The name 'LONG_TERM_DISCOUNT' could be more descriptive. Since it's a discount for rentals of 7 or more days, consider renaming it to 'DISCOUNT_FOR_7_DAYS_OR_MORE' to clearly reflect its purpose.